imapclient/myrights.go

```diff
@@ -0,0 +1,49 @@
+package imapclient
+
+import (
+	"fmt"
+
+	"github.com/emersion/go-imap/v2"
+	"github.com/emersion/go-imap/v2/internal/imapwire"
+)
+
+// Myrights sends a MYRIGHTS command.
+func (c *Client) Myrights(mailbox string) *MyrightsCommand {
```

Comment:
Nit: I'd find the capitalization `MyRights` to be more readable.

Reply:
fixed